### ISSUE TITLE
Enable JSON/XML output for missions

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-server#523]
- http://world.openfoodfacts.org/mission/first-contribution.json
- http://world.openfoodfacts.org/missions.json

**Comment by hangy** [comment]
Maybe after #284? Currently, the missions are not of that much use.

**Comment by teolemon** [comment]
It is for use in the mobile app, for existing missions.  Not a priority though, I guess.

**Comment by teolemon** [comment]
Blocking Android app bug > https://github.com/openfoodfacts/OpenFoodFacts-androidApp/issues/420

**Comment by hangy** [comment]
As far as I know, the old missions have been deprecated.
